src/main/resources/db/changelog/changelog.yml

```diff
@@ -0,0 +1,124 @@
+databaseChangeLog:
+  - changeSet:
+      id: 1
+      author: Liquibase
+      changes:
+        - createTable:
```

Comment:
please create the separate `.sql` file for any changes you add to the database
this is an example of `changelog.yml` file
```
databaseChangeLog:
  - changeSet:
      runAlways: false
      id: 20201026_01 // actually here could be any id, but here the date is used
      author: Alexey Korovko
      changes:
        - sqlFile:
            path: sql/001-create-registered-user.sql
            enchoding: utf8
            relativeToChangelogFile: true
```

and this is `001-create-registered-user.sql` file:
```
set search_path = public;

CREATE TABLE registered_user
(
    id                      uuid                    NOT NULL,
    first_name              text                    NOT NULL,
    last_name               text                    NOT NULL
....
);
ALTER TABLE registered_user ADD CONSTRAINT registered_user_pkey PRIMARY KEY (id);
```

so, for any changes you do to your database you need to create separate `.sql` file and add a record about this into `changelog.yml`